CRM/Contact/Page/View/Summary.php

```diff
@@ -243,7 +266,6 @@ public function view() {
       }
     }
 
-    $defaults['external_identifier'] = $contact->external_identifier;
```

Comment:
moved up, will have actual value added by `CRM_Contact_BAO_Contact::retrieve` like the others